src/AWS.Deploy.Recipes/CdkTemplates/ConsoleAppECSFargateScheduleTask/Generated/Recipe.cs

```diff
@@ -0,0 +1,183 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+
+using Amazon.CDK;
+using Amazon.CDK.AWS.EC2;
+using Amazon.CDK.AWS.ECS;
+using Amazon.CDK.AWS.ECR;
+using Amazon.CDK.AWS.ECS.Patterns;
+using Amazon.CDK.AWS.IAM;
+using AWS.Deploy.Recipes.CDK.Common;
+
+using ConsoleAppECSFargateScheduleTask.Configurations;
+
+using Protocol = Amazon.CDK.AWS.ECS.Protocol;
+using Schedule = Amazon.CDK.AWS.ApplicationAutoScaling.Schedule;
+
+// This is a generated file from the original deployment recipe. It is recommended to not modify this file in order
+// to allow easy updates to the file when the original recipe that this project was created from has updates.
+// To customize the CDK constructs created in this file you should use the AppStack.CustomizeCDKProps() method.
+
+namespace ConsoleAppECSFargateScheduleTask
+{
+    using static AWS.Deploy.Recipes.CDK.Common.CDKRecipeCustomizer<Recipe>;
+
+    public class Recipe : Construct
+    {
+        public IRole? AppIAMTaskRole { get; set; }
+
+        public IVpc? AppVpc { get; set; }
+
+        public ICluster? EcsCluster { get; set; }
+
+        public ContainerDefinitionOptions? AppContainerDefinition { get; set; }
+
+        public AwsLogDriver? AppLogging { get; set; }
+
+        public FargateTaskDefinition? AppTaskDefinition { get; set; }
+
+        public ScheduledFargateTask? ScheduleTask { get; set; }
+
+        public Recipe(Construct scope, IRecipeProps<Configuration> props)
+            : base(scope, "Recipe")
```

Comment:
There should be a comment explaining why the construct id is constant here. And changing it will impact the stack.